init.lua

```diff
@@ -157,34 +157,27 @@ minetest.register_abm({
 -- Compatibility with stairsplus
 --
 
-if minetest.get_modpath("moreblocks") and settings:get_bool("ENABLE_STAIRSPLUS") then
-	register_stair_slab_panel_micro("quartz", "block", "quartz:block",
-		{cracky=3},
-		{"quartz_block.png"},
-		"Quartz Block",
-		"block",
-		0
-	)
-
-	register_stair_slab_panel_micro("quartz", "chiseled", "quartz:chiseled",
-		{cracky=3},
-		{"quartz_chiseled.png"},
-		"Chiseled Quartz",
-		"chiseled",
-		0
-	)
-
-	register_stair_slab_panel_micro("quartz", "pillar", "quartz:pillar",
-		{cracky=3},
-		{"quartz_pillar_top.png", "quartz_pillar_top.png", "quartz_pillar_side.png"},
-		"Quartz Pillar",
-		"pillar",
-		0
-	)
-
-	table.insert(circular_saw.known_stairs, "quartz:block")
-	table.insert(circular_saw.known_stairs, "quartz:chiseled")
-	table.insert(circular_saw.known_stairs, "quartz:pillar")
+if minetest.global_exists("stairsplus") then
```

Comment:
add `stairsplus` as optional dependency. Also should be mentioned in the readme?

Reply:
I thought `moreblocks` included `stairsplus` (it adds it as a global anyway), but I'll add `stairsplus` anyway.

Reply:
Fixed.